### PR TITLE
release 0.0.1

### PR DESCRIPTION
**Pull request by chfw**
for all what we have in the past and the recent past, let's now version it as 0.0.1

### CODE REVIEW

**Comment by chfw** [comment]
there has been zig-zag lines in commits but they are done and dusted. there exists a separate repo that ships pypi-mobans as python package, in supporting yehua.

File: .moban.dt/local-README.rst.jj2 [hunk 1]
```diff
@@ -29,6 +31,14 @@ setup.py
 {% endblock %}
 
 {% block bottom_block %}
+Development process
+================================================================================
+
+Please fork and make pull request to **dev** branch. Per each release, dev branch
+will be merge into master branch.
```
Comment (ayan-b) [inline .moban.dt/local-README.rst.jj2]: merge --> merged

File: .moban.dt/local-README.rst.jj2 [hunk 2]
```diff
@@ -8,8 +8,10 @@
 It is used with `yehua <https://github.com/chfw/yehua>`_.
 Organisations using it:
 
-- `pyexcel <https://github.com/pyexcel/pyexcel>`_.
-- `coala <https://github.com/coala/coala>`_.
+- `pyexcel mobans <https://github.com/pyexcel/pyexcel-mobans>`_.
+- `coala mobans<https://gitlab.com/coala/mobans>`_.
```
Comment (ayan-b) [inline .moban.dt/local-README.rst.jj2]: A space is missing between `mobans` and `<`. Links will not properly render. ([Preview](https://github.com/moremoban/pypi-mobans/blob/825fc08af2c993673030163e901dccb1b26cfd01/README.rst))
```suggestion
- `coala mobans <https://gitlab.com/coala/mobans>`_.
```

File: .moban.dt/local-README.rst.jj2 [hunk 3]
```diff
@@ -8,8 +8,10 @@
...
+- `pyecharts pypkg mobans <https://github.com/pyecharts/pypkg-mobans>`.
+- `echarts maps mobans <https://github.com/echarts-maps/echarts-js-mobans>`.
```
Comment (ayan-b) [inline .moban.dt/local-README.rst.jj2]: An underscore is missing. Links will not properly render.
```suggestion
- `pyecharts pypkg mobans <https://github.com/pyecharts/pypkg-mobans>`_.
- `echarts maps mobans <https://github.com/echarts-maps/echarts-js-mobans>`_.
```

File: .moban.dt/local-README.rst.jj2 (hunk 1 above)
Comment (chfw) [reply .moban.dt/local-README.rst.jj2]: sharp eyes

File: .moban.dt/local-README.rst.jj2 (hunk 2 above)
Comment (chfw) [reply .moban.dt/local-README.rst.jj2]: true

**Pull request merged**